arm-commerce/2015-06-01-preview/swagger/commerce.json

```diff
@@ -116,7 +116,7 @@
             "in": "query",
             "required": false,
             "type": "string",
-            "description": "The filter to apply on the operation. It ONLY supports the �eq� and �and� logical operators at this time. All the 4 query parameters 'OfferDurableId',  'Currency', 'Locale', 'Region' are required to be a part of the $filter."
```

Comment:
Yes, I changed both of the quotes. But the diff generator thinks the _word_ itself changed, which it didn't. I checked in the commit diff; it's just a GitHub display issue.